server/src/main/java/org/elasticsearch/index/query/QueryRewriteContext.java

```diff
@@ -124,4 +124,11 @@ public void onFailure(Exception e) {
         }
     }
 
+    /**
+     * @return whether the query rewrite context supports 'now' (current time) in range queries with data ranges.
+     */
+    public boolean supportsNowInRangeQueries() {
```

Comment:
maybe `ignoresNowInRangeQueries` since we only rewrite the range to a `match_all`  ?

Reply:
`convertNowRangeToMatchAll()` ?